desktop/linux/index.md

```diff
@@ -144,7 +160,12 @@ $ sudo apt remove docker-desktop
 
 ## Known issues
 
- - The **Reset to factory defaults** option on the **Troubleshoot** page currently does not work.
+ - Docker CLI login flow presents some inconsistencies that are under investigation. If problems are encountered when trying to log in, remove the `credsStore` property from `~/.docker/config.json` and restart Docker Desktop (run either 
```

Comment:
```suggestion
 - The Docker CLI login flow has some inconsistencies that we are currently investigation. If you experience any issues when trying to log in, remove the `credsStore` property from `~/.docker/config.json` and restart Docker Desktop (run either 
```

Reply:
If you remove `credStore` property, you will need to close the dashboard and reopen it to get in sync with the cli state.